README.md

```diff
@@ -1,135 +1,168 @@
 
-With the Gemini CLI you can:
+## 🚀 Why Gemini CLI?
 
-- Query and edit large codebases in and beyond Gemini's 1M token context window.
-- Generate new apps from PDFs or sketches, using Gemini's multimodal capabilities.
-- Automate operational tasks, like querying pull requests or handling complex rebases.
-- Integrate with GitHub: Use the [Gemini CLI GitHub Action](https://github.com/google-github-actions/run-gemini-cli) for automated PR reviews, issue triage, and on-demand AI assistance directly in your repositories.
-- Use tools and MCP servers to connect new capabilities, including [media generation with Imagen,
-  Veo or Lyria](https://github.com/GoogleCloudPlatform/vertex-ai-creative-studio/tree/main/experiments/mcp-genmedia)
-- Ground your queries with the [Google Search](https://ai.google.dev/gemini-api/docs/grounding)
-  tool, built into Gemini.
+- **🎯 Free tier**: 60 requests/min and 1,000 requests/day with personal Google account
+- **🧠 Powerful Gemini 2.5 Pro**: Access to 1M token context window
+- **🔧 Built-in tools**: Google Search grounding, file operations, shell commands, web fetching
+- **🔌 Extensible**: MCP (Model Context Protocol) support for custom integrations
+- **💻 Terminal-first**: Designed for developers who live in the command line
+- **🛡️ Open source**: Apache 2.0 licensed
 
-## Quickstart
+## 📦 Installation
 
-You have two options to install Gemini CLI.
+### Quick Install
 
-### With Node
+#### Run instantly with npx
 
-1. **Prerequisites:** Ensure you have [Node.js version 20](https://nodejs.org/en/download) or higher installed.
-2. **Run the CLI:** Execute the following command in your terminal:
-
-   ```bash
-   npx https://github.com/google-gemini/gemini-cli
-   ```
-
-   Or install it with:
-
-   ```bash
-   npm install -g @google/gemini-cli
-   ```
-
-   Then, run the CLI from anywhere:
-
-   ```bash
-   gemini
-   ```
-
-### With Homebrew
-
-1. **Prerequisites:** Ensure you have [Homebrew](https://brew.sh/) installed.
-2. **Install the CLI:** Execute the following command in your terminal:
-
-   ```bash
-   brew install gemini-cli
-   ```
-
-   Then, run the CLI from anywhere:
+```bash
+# Using npx (no installation required)
+npx https://github.com/google-gemini/gemini-cli
+```
 
-   ```bash
-   gemini
-   ```
+#### Install globally with npm
+```bash
+npm install -g @google/gemini-cli
+```
 
-### Common Configuration steps
+#### Install globally with Homebrew (macOS/Linux)
+```bash
+brew install gemini-cli
+```
 
-3. **Pick a color theme**
-4. **Authenticate:** When prompted, sign in with your personal Google account. This will grant you up to 60 model requests per minute and 1,000 model requests per day using Gemini.
+#### System Requirements
+- Node.js version 20 or higher
+- macOS, Linux, or Windows
+
+## 📋 Key Features
+
+With Gemini CLI you can:
+
+- **Code Understanding & Generation**
```

Comment:
This would also be "Code understanding and generation:" 

But, in the interest of expedience, I think an option is simply submitting this PR and I can go back through for small things like style.